x11docker

```diff
@@ -935,6 +935,7 @@ cleanup() {             # --cleanup : check for non-removed containers and left
   grep -q .cache/x11docker <<<$Cachebasefolder && Orphanedfolders=$(find "$Cachebasefolder" -mindepth 1 -maxdepth 1 -type d | sed s%$Cachebasefolder/%% | grep -v x11docker-gui)
   [ "${X11DOCKER_LASTCLEANFOLDER:-}" ] && Orphanedfolders="$(echo "$Orphanedfolders" | grep -v $X11DOCKER_LASTCLEANFOLDER)"
   Orphanedcontainers="$(docker ps -a --filter name=x11docker_X --format "{{.Names}}")"
+  Orphanedcontainers="$Orphanedcontainers $(find "$Cachebasefolder" -mindepth 1 -maxdepth 1 -type f -name 'container.*' | sed 's/^.*\.//')"
```

Comment:
- Misunderstanding: I thought of a file container.name that contains the name of the container.
 - Bug: find will also find files like container.id and container.pid; they would become part of $Orphanedcontainers.
 - Solution: don't use container.name or container.somecustomname. Instead, use content of already existing container.id.